amico/core.py

```diff
@@ -224,7 +247,7 @@ def load_kernels( self ) :
         idx_OUT, Ylm_OUT = amico.lut.aux_structures_resample( self.scheme, self.get_config('lmax') )
 
         # Dispatch to the right handler for each model
-        self.KERNELS = self.model.resample( self.get_config('ATOMS_path'), idx_OUT, Ylm_OUT )
+        self.KERNELS = self.model.resample( self.get_config('ATOMS_path'), idx_OUT, Ylm_OUT, self.get_config('doMergeB0') )
```

Comment:
This breaks the code in COMMIT, see [here](https://github.com/daducci/COMMIT/blob/master/commit/core.py#L219). Can we also apply this strategy of delegating to the `model.resample` the stripping of the b0s in the `KERNELS` structure?